src/components/Button/Button.style.ts

```diff
@@ -232,6 +232,7 @@ function buttonVariants({
   format,
   theme,
   variant,
+  active,
```

Comment:
Can this be moved in the top for alphabetical order?

Reply:
moved